src/RestInterface/RestInterface.cpp

```diff
@@ -44,112 +44,90 @@ namespace supra
 	using namespace std;
 	using namespace logging;
 	
-	void RestInterface::initRestOpHandlers() {
+	void RestInterface::initRestOpHandlers()
+	{
 		_listener.support(methods::GET, std::bind(&RestInterface::handleGet, this, std::placeholders::_1));
 		_listener.support(methods::PUT, std::bind(&RestInterface::handlePut, this, std::placeholders::_1));
 		_listener.support(methods::POST, std::bind(&RestInterface::handlePost, this, std::placeholders::_1));
 		_listener.support(methods::DEL, std::bind(&RestInterface::handleDelete, this, std::placeholders::_1));
 		_listener.support(methods::PATCH, std::bind(&RestInterface::handlePatch, this, std::placeholders::_1));
 	}
 
-	void RestInterface::handleGet(http_request message) {
+	void RestInterface::handleGet(http_request message)
+	{
 		auto path = requestPath(message);
 		if (!path.empty()) 
-		{
-			/*if (path[0] == "service" && path[1] == "test") {
-				auto response = json::value::object();
-				response["version"] = json::value::string("0.1.1");
-				response["status"] = json::value::string("ready!");
-				message.reply(status_codes::OK, response);
-			}*/
-			
+		{			
 			if (path[0] == "nodes") 
 			{
-				auto nodeType = path[1];
+				std::string nodeType; 
 				if (path.size() == 1)
-                {
-                    // MARK: Get request to /nodes is enough to get all the nodes
-                    auto nodeType = "all";
-                    auto nodeIDs = get_nodes(nodeType);
-                    json::value responseIDs = json::value::array();
-
-                    for (size_t k = 0; k < nodeIDs.size(); k++)
-                    {
-                        responseIDs[k] = json::value::string(nodeIDs[k]);
-                    }
-                    auto response = json::value::object();
-                    response["nodeIDs"] = responseIDs;
-                    message.reply(status_codes::OK, response);
-                }
-				else if (path.size()>1)
-                {
-                    auto nodeType = path[1];
-                    auto nodeIDs = get_nodes(nodeType);
-                    json::value responseIDs = json::value::array();
-
-                    for (size_t k = 0; k < nodeIDs.size(); k++) {
-                        responseIDs[k] = json::value::string(nodeIDs[k]);
-                    }
-
-                    auto response = json::value::object();
-                    response["nodeIDs"] = responseIDs;
-                    message.reply(status_codes::OK, response);
-                }
-								
+				{
+					// MARK: Get request to /nodes is enough to get all the nodes
+					nodeType = "all";
```

Comment:
nodeType here doesn't get used, and requests to /nodes won't return anything. The code from before was adding this functionality which you have removed.